templates/addons/metrics-server/metrics-server.yaml

```diff
@@ -135,7 +138,7 @@ spec:
         - --kubelet-use-node-status-port
         - --metric-resolution=15s
         - --kubelet-insecure-tls
-        image: k8s.gcr.io/metrics-server/metrics-server:v0.5.2
+        image: k8s.gcr.io/metrics-server/metrics-server:v0.6.2
```

Comment:
#3035 recommends that we change the base URL here to registry.k8s.io (from k8s.gcr.io). So this would be `registry.k8s.io/metrics-server/metrics-server:v0.6.2`.

Is that correct? Let me test it:

```
% docker pull registry.k8s.io/metrics-server/metrics-server:v0.6.2
v0.6.2: Pulling from metrics-server/metrics-server
76c00402984b: Pull complete 
ae5a0379bc32: Pull complete 
Digest: sha256:f977ad859fb500c1302d9c3428c6271db031bb7431e7076213b676b345a88dc2
Status: Downloaded newer image for registry.k8s.io/metrics-server/metrics-server:v0.6.2
registry.k8s.io/metrics-server/metrics-server:v0.6.2
```

Yes, looks good.